blivet/formats/luks.py

```diff
@@ -250,7 +250,7 @@ def _create(self, **kwargs):
     def _post_create(self, **kwargs):
         super(LUKS, self)._post_create(**kwargs)
         self.uuid = blockdev.crypto.luks_uuid(self.device)
-        if flags.installer_mode or not self.map_name:
+        if not self.map_name:
```

Comment:
You're probably going to have to unset `self.map_name` in `LUKS._pre_create` as I think it might default to `luks-$(basename $device)`.